Vagrantfile

```diff
@@ -93,7 +93,6 @@ EOF
     export osbuilder_distro="$distro"
 
     cd ${kata_tests_repo_dir}
-    sudo -E PATH=$PATH -H -u #{guest_user} bash -c '.ci/setup.sh'
```

Comment:
This seems like it definitely needs a docs update, because if you create a Vagrant vm directly (which you might want to do for some more specific test not covered by the existing jobs) it will no longer be set up to do that as is.